ExtendedDescription.md

```diff
@@ -1,121 +1,154 @@
-* Exercise consideration and respect in your speech and actions.
-* Attempt collaboration & de-escalation before conflict.
-* If you inadvertently cause offence to someone else, take responsibility for your actions.
-* Refrain from demeaning, discriminatory, or harassing behavior and speech.
-* Be mindful of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
-  
-### Unacceptable Behavior
+### Declaimers and Limitation on Liability
 
-To be specific, the following behaviors are unacceptable within our community:
+The nelson.social online space is provided “as is” and “as available.” We disclaim all conditions, representations and warranties not expressly set out in these terms to the fullest extent permitted by law.
 
-* Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in British Columbia or Canada.
-* Violence, threats of violence or violent language directed against another person.
-* Using slurs or racist, sexist, homophobic, transphobic, ableist or otherwise discriminatory or hateful jokes or language; promoting white supremacy, anti-Semitism, transphobia, castism or other hateful ideologies.
-* Posting or displaying sexually explicit or violent material.
-* Posting or threatening to post other people's personally identifying information ("doxing").
-* Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
-* Unwelcome sexual attention. This includes, sexualized comments or jokes; unwelcomed sexual advances.
-* Deliberate intimidation, stalking or following (online or in person).
-* Debating moderation actions in public. If you disagree with how a moderator enforced this Code of Conduct you may direct-message the moderation team.
-* Tone policing. Messages you see on nelson.social may violate the standards of civility we set on nelson.social, but you should not criticize users for their tone. If the user is a member of the nelson.social community, you should report them; if not, you should feel free to block them. In neither case should you criticize their tone.
-* Advocating for, or encouraging, any of the above behavior.
+We shall not be liable for delay or failure in performance for causes beyond our control or any other
+damage which does not result from a breach of our obligations under the Terms of Use.
 
-Again, we suggest you [find another server](https://instances.social/) or even [set up your own](https://runyourown.social/) if you don't agree with these rules.
+We are not liable for business losses. We only the nelson.social online space for your personal, non-commercial, and domestic use. If you use our online space for any other purpose we will have no liability to you for any loss of profit, loss of business, business interruption, loss of business opportunity, or similar loss.
```

Comment:
"We only the" - there seems to be a word missing here?

Reply:
Will confirm.